pages/jutsu.php

```diff
@@ -188,10 +188,12 @@ function jutsu(): void {
 
     if($jutsu_to_view == null) {
         $sortedJutsu = [];
-        foreach($player->ninjutsu_ids as $jutsu_id) {
-            $sortedJutsu[] = $player->jutsu[$jutsu_id]->rank;
+        if ($player->ninjutsu_ids) {
+            foreach($player->ninjutsu_ids as $jutsu_id) {
+                $sortedJutsu[] = $player->jutsu[$jutsu_id]->rank;
+            }
+            array_multisort($sortedJutsu, $player->ninjutsu_ids);
```

Comment:
Hmm... What if we go the other way with this and change the definition to initialize `ninjutsu_ids` (and tai/gen) as empty arrays? That seems simpler to use, and we can still check `count($player->ninjutsu_ids)` if desired